src/Runner/Job.php

```diff
@@ -245,4 +245,24 @@ public function getHeaders()
 		return $this->headers;
 	}
 
+
+	/**
+	 * @return string
+	 */
+	private function buildArgumentsList()
+	{
+		$args = [];
+
+		foreach ($this->args as $key => $arg) {
```

Comment:
@jiripudil The `key=val` is the Tester's "internal" format. Someone else can append own args from outside.